saasherder/cli.py

```diff
@@ -2,8 +2,8 @@
 
 import argparse
 
-from saasherder import SaasHerder
-from config import SaasConfig
+from .saasherder import SaasHerder
```

Comment:
This (and other changes like this one) should not be part of this PR - it is not relevant for the change, please push it as a separate PR